website/content/docs/agent/caching/persistent-caches/kubernetes.mdx

```diff
@@ -0,0 +1,26 @@
+---
+layout: docs
+page_title: Vault Agent Kubernetes Persistent Cache
+sidebar_title: Kubernetes
+description: Kubernetes Persistent Cache for Vault Agent Caching
+---
+
+# Vault Agent Kubernetes Persistent Cache
+
+When `kubernetes` is configured for the persistent cache type, Vault Agent will optimize the 
+persistent cache specifically for Kubernetes. This type of persistent cache requires a Kubernetes 
+service account token. The service account token is used during encryption and decryption of the 
+persistent cache as an additional integrity check.
+
+The Vault Agent persistent cache file in Kubernetes should only be used for handing off Vault tokens 
+and leases between initialization and sidecar Vault Agent containers. This cache file should be shared 
+using a memory volume between the Vault Agent containers.
+
+If the Vault Agent Injector for Kubernetes is being used, the persistent cache is automatically configured 
+and used if the annotation `vault.hashicorp.com/agent-cache-enable: true` is set.
```

Comment:
Maybe this should link to the annotation doc? I think this might work?
```suggestion
and used if the annotation [`vault.hashicorp.com/agent-cache-enable: true`](/docs/platform/k8s/injector/annotations#vault-hashicorp-com-agent-cache-enable) is set.
```